AspNetCore.sln

```diff
@@ -1618,6 +1618,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorWinFormsApp", "src\Co
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Http.Abstractions.Microbenchmarks", "src\Http\Http.Abstractions\perf\Microbenchmarks\Microsoft.AspNetCore.Http.Abstractions.Microbenchmarks.csproj", "{3F752B48-2936-4FCA-B0DC-4AB0F788F897}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.EToETests", "src\Components\test\EToETest\Microsoft.AspNetCore.Components.EToETests.csproj", "{60B82C36-E613-496D-AD1B-0015DDAEFD96}"
```

Comment:
How about `PlayweightE2ETests` so it's clear how they are different?

Reply:
Gamer in me can't help but think P2W with this particular combination of letters  P2W.Tests

Reply:
Let's not put other product names in test projects. Those things change and become stale. We can call them Components.FunctionalTests or Component.EndToEnd or a similar generic name. In the end, if/when the migration is successful we can delete the old project and name the new project Components.E2ETests again if we want.

Reply:
Should we make a specific goal of eventually removing the old `Components.E2ETests` and having the new one take that name? That would make sense to me.

In the meantime, it would be great for the new project to have some name that indicates why it exists. It will be really hard for anyone coming fresh to the repo to understand why we have both `E2ETests` and `EToETests`. How about calling the new one `E2ETestMigration` or similar, with the goal that we eventually replace the old one and rename it?

Reply:
Sounds good, will start a new PR since this has gotten stale, with rebased from main, with E2ETestMigration as the folder, and  the csproj will be the same just with an additional `Components.Migration.E2ETests.csproj`